docs/getting-started/business/splunk-app.md

```diff
@@ -54,8 +54,11 @@ in Splunk.
    docker run --rm --platform linux/amd64 --name splunk -d -p 8001:8000 -p 8089:8089 -e SPLUNK_START_ARGS='--accept-license' -e SPLUNK_PASSWORD='password' splunk/splunk:9.3
    ```
 
-   Please note this will set the admin password to `password`. This is for development purposes
-   only.
+:::warning If you are using an Apple Silicon Mac, you must use up to version 9.3 of splunk. As of
+version 9.4, splunk depends on the use of the AVX instruction set for its KVStore, which is not
+supported by Apple Silicon. :::
```

Comment:
Copy edits for:
- formatting for admonitions (the "warning" box) - note you need an empty line between paragraphs for the linter not to collapse it
- capital leter for Splunk

```suggestion
:::warning

If you are using an Apple Silicon Mac, you must use up to version 9.3 of Splunk. As of
version 9.4, Splunk requires the AVX instruction set for its KVStore, which is not
supported by Apple Silicon.

:::
```

(This might need a run through the linter to fix linebreaks.)